website/markdown/posts/2020-12-24-next-for-tuist/post.mdx

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Next for Tuist
+categories: ['Tuist', '2021', 'vision']
+excerpt: It's a wrap for 2020. In this blog post we share the vision of Tuist for 2021. We'll bring support for plug-ins, improve and standardize the integration of third party dependencies, add support for selective building and testing, and much more.
```

Comment:
I believe plug-in and plugin are both acceptable, but plug-in in this context seems a little bit odd to me? But maybe it's just me 😁

Reply:
Yeah, I was originally inclined towards plugins, but I ended up changing it for plugins because I had seen it in some other places written like that. I changed it to plugin again.